docs/src/main/sphinx/release/release-419.md

```diff
@@ -0,0 +1,33 @@
+# Release 419 (24 May 2023)
+
+## General
+
+* Add the {func}`array_histogram` function to find the number of occurrences of
+  the unique elements in an array. ({issue}`14725 `)
+* Fix failure when processing string columns with values larger than 5 MB. ({issue}`17557`)
```

Comment:
this is inaccurate and downplays the cases where it can happen. It can happen anytime the server sends the client a response with a string value larger than 5MB. The string value doesn't have to be a value from the table, it can even be things like query stats that we send.

IMO the wording here should be to avoid all Trino JDBC and Trino CLI >= 415 and < 419.

Reply:
It's just easiest to reproduce with a table having large string values but that's only one of the possible failures.

Reply:
Since it applies to deserialization path I'd phrase it like: `fix query failures when server JSON responses exceed 5MB length limit for string values`